packages/xo-web/src/common/intl/messages.js

```diff
@@ -866,6 +868,9 @@ const messages = {
   editHostIscsiIqnMessage:
     'Are you sure you want to edit the iSCSI IQN? This may result in failures connecting to existing SRs if the host is attached to iSCSI SRs.',
   hostTitleRamUsage: 'Host RAM usage:',
+  maintenanceHostModaleMessage: 'This disables the host and potentially migrates a lot of VMs.',
```

Comment:
This wasn't a suggestion for the modal message but my explanation of why we need a modal :) Let's keep the message with the same format as the other ones, something like: "Are you sure you want to enter maintenance mode? This will migrate all the VMs running on this host to other hosts of the pool."